ibc-testkit/src/testapp/ibc/core/types.rs

```diff
@@ -531,94 +293,56 @@ where
         self
     }
 
-    pub fn with_client_config(mut self, client: MockClientConfig) -> Self {
-        let cs_heights = if client.consensus_state_heights.is_empty() {
-            vec![client.latest_height]
+    pub fn generate_light_client(
```

Comment:
Would it be beneficial to add a docstring for this. Does this method generate a client of the counterparty context within the current one, or is it the other way around?

Reply:
Make sense. Here I create the light client of the local context. This should go under `RelayerContext`. Let's tackle this in the next PR.